packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -430,21 +433,15 @@ export class ElectronMainApplication {
     }
 
     protected async getForkOptions(): Promise<ForkOptions> {
-        return {
-            env: {
-                ...process.env,
-                [ElectronSecurityToken]: JSON.stringify(this.electronSecurityToken),
-            },
-        };
+        return { env: process.env };
```

Comment:
I think you have changed the behavior, haven't you?
```ts
let process = { env: { foo: 'bar' } };
const obj1 = { env: process.env };
obj1.env.foo = 'baz';
console.log(process.env.foo === 'bar', process.env.foo); // false, "baz"

process = { env: { foo: 'bar' } };
const obj2 = { env:  { ...process.env } };
obj2.env.foo = 'baz';
console.log(process.env.foo === 'bar', process.env.foo); // true, "bar"
```

To avoid such issues, I would like to ask you to revert all the unnecessary changes you have made in this PR (e.g., changing the order or the imports, order of the bindings in the module, cosmetic things) to simplify the review process for us. Thanks!

Once you decided how to move on with this PR, please ping me; I am happy to verify/review the electron part.

Let me know if you disagree.

Reply:
For the env, yes I changed the way it worked to always stuff the `ElectronSecurityToken` into electron main's env. This is what we did when loading the backend code into that same process (for debugging), and just applied the same strategy for all cases. This is not cosmetic, this is part of refactoring those things into the new contributions.

For the imports, sorry it got autoformatted by organize them. Will revert those.

Reply:
Reverted this because I don't want to spend time on this specific issue, I also removed some noise from the PR.